public class java.lang.ClassNotFoundException extends java.lang.ReflectiveOperationException
{

    public void <init>()
    {
        java.lang.ClassNotFoundException r0;
        java.lang.RuntimeException $r1;

        r0 := @this: java.lang.ClassNotFoundException;

        specialinvoke r0.<java.lang.ReflectiveOperationException: void <init>()>();

        $r1 = new java.lang.RuntimeException;

        specialinvoke $r1.<java.lang.RuntimeException: void <init>(java.lang.String)>("Stub!");

        throw $r1;
    }

    public void <init>(java.lang.String)
    {
        java.lang.ClassNotFoundException r0;
        java.lang.String r1;
        java.lang.RuntimeException $r2;

        r0 := @this: java.lang.ClassNotFoundException;

        r1 := @parameter0: java.lang.String;

        specialinvoke r0.<java.lang.ReflectiveOperationException: void <init>()>();

        $r2 = new java.lang.RuntimeException;

        specialinvoke $r2.<java.lang.RuntimeException: void <init>(java.lang.String)>("Stub!");

        throw $r2;
    }

    public void <init>(java.lang.String, java.lang.Throwable)
    {
        java.lang.ClassNotFoundException r0;
        java.lang.String r1;
        java.lang.Throwable r2;
        java.lang.RuntimeException $r3;

        r0 := @this: java.lang.ClassNotFoundException;

        r1 := @parameter0: java.lang.String;

        r2 := @parameter1: java.lang.Throwable;

        specialinvoke r0.<java.lang.ReflectiveOperationException: void <init>()>();

        $r3 = new java.lang.RuntimeException;

        specialinvoke $r3.<java.lang.RuntimeException: void <init>(java.lang.String)>("Stub!");

        throw $r3;
    }

    public java.lang.Throwable getException()
    {
        java.lang.ClassNotFoundException r0;
        java.lang.RuntimeException $r1;

        r0 := @this: java.lang.ClassNotFoundException;

        $r1 = new java.lang.RuntimeException;

        specialinvoke $r1.<java.lang.RuntimeException: void <init>(java.lang.String)>("Stub!");

        throw $r1;
    }

    public java.lang.Throwable getCause()
    {
        java.lang.ClassNotFoundException r0;
        java.lang.RuntimeException $r1;

        r0 := @this: java.lang.ClassNotFoundException;

        $r1 = new java.lang.RuntimeException;

        specialinvoke $r1.<java.lang.RuntimeException: void <init>(java.lang.String)>("Stub!");

        throw $r1;
    }
}
